apps/condo/pages/incident/index.tsx

```diff
@@ -419,3 +427,20 @@ const IncidentsPage: IIncidentIndexPage = () => {
 IncidentsPage.requiredAccess = IncidentReadPermissionRequired
 
 export default IncidentsPage
+
+export const getServerSideProps: GetServerSideProps = async (context) => {
```

Comment:
Why not put this into one common function and reuse it?
Like this:
```
export const getServerSideProps: GetServerSideProps = getDefaultServerSideProps
```

Reply:
One line - context destructuring
Two lines - get header and client
Two lines - prefetch auth
One line - prefetch employees
One line - return ssr state

From all this I can only take out into a common function only prefetch auth and prefetch employee (3 lines). I don't see much use in this :)

in the following pull requests we will supplement this logic on the pages